### DISCUSSION TITLE
Replays

### DISCUSSION THREAD

**Discussion by osuplayer73** [ppy/osu#17521] [Ideas] [answered]
I Don't know if this has been suggested already (tbh probably) but here it is anyways.
whilst playing osu! There are a lot of times where i fail a certain section and i'm really curious as too why that happened / wanting to know what i did wrong so i end up going back and watching my replay and tbh, this may sound "bad"? But having to wait to sometimes 1-2m before reaching that certain pattern is pretty annoying... So i've been looking on the internet to see if there were programs that could help me with that. And there is! It's called "Rewind". 

Here is the suggestion / question : Could it be possible to add more depth into the "replay" section? Without needing to open an exterior application for it to be possible (which im pretty sure Rewind doesn't work for !lazer)

Rewind : 
-Lets you slow down the entire map 0.25x .5x .75x
-Shows you exactly where your cursor has been clicked
-Lets you select exact time frames with the help of a bar at the bottom
-Allows you to set custom mods to that replay 

This is it, like i said i know this does sound like a "im too lazy to" so i would understand this request being "denied" :P 
But i thought why not give it a shot? And also maybe the fact that there are already exterior programs that makes this possible could another reason why this wouldn't be "necessary" 

thanks for reading and like i said, sorry if this was already requested

**Comment by peppy** [comment, marked as answer]
Has been suggested countless times before, and will definitely be implemented. Note that the replay viewer already allows seeking to point in time, so the main blocker here is the UX of this flow, and ability to watch failed plays.

**Reply by osuplayer73** [reply]
That is great to hear! thanks for the great communication, its always a pleasure hearing your thoughts :)
